.gitignore

```diff
@@ -1,3 +1,5 @@
 /.vagrant
 /node_modules
 package-lock.json
+.env
+config.js
```

Comment:
Add a newline after the file here.

Reply:
Also I think we should let a *template* `config.js` be included in this commit and then take put it down in the `gitignore`. Because then we have no idea what `env` variables you have used throughout the code.